eng/common/scripts/copy-docs-to-blobstorage.ps1

```diff
@@ -257,30 +259,30 @@ if ($Language -eq "javascript")
 
 if ($Language -eq "dotnet")
 {
-    $PublishedPkgs = Get-ChildItem "$($DocLocation)/packages" | Where-Object -FilterScript {$_.Name.EndsWith(".nupkg") -and -not $_.Name.EndsWith(".symbols.nupkg")}
-    $PublishedDocs = Get-ChildItem "$($DocLocation)" | Where-Object -FilterScript {$_.Name.StartsWith("Docs.")}
+    $PublishedPkgs = Get-ChildItem "$($DocLocation)" | Where-Object -FilterScript {$_.Name.EndsWith(".nupkg") -and -not $_.Name.EndsWith(".symbols.nupkg")}
+    $PublishedDocs = Get-ChildItem "$($DocLocation)" | Where-Object -FilterScript {$_.Name.EndsWith("docs.zip")}
 
-    foreach ($Item in $PublishedDocs) {
-        $PkgName = $Item.Name.Remove(0, 5)
-        $PkgFullName = $PublishedPkgs | Where-Object -FilterScript {$_.Name -match "$($PkgName).\d"}
-
-        if (($PkgFullName | Measure-Object).count -eq 1)
-        {
-            $DocVersion = $PkgFullName[0].BaseName.Remove(0, $PkgName.Length + 1)
-
-            Write-Host "Start Upload for $($PkgName)/$($DocVersion)"
-            Write-Host "DocDir $($Item)"
-            Write-Host "PkgName $($PkgName)"
-            Write-Host "DocVersion $($DocVersion)"
-            $releaseTag = RetrieveReleaseTag "Nuget" $PublicArtifactLocation 
-            Upload-Blobs -DocDir "$($Item)" -PkgName $PkgName -DocVersion $DocVersion -ReleaseTag $releaseTag
-        }
-        else
-        {
-            Write-Host "Package with the same name Exists. Upload Skipped"
-            continue
-        }
+    if (($PublishedPkgs.Count -gt 1) -or ($PublishedDoc.Count -gt 1))
+    {
+        Write-Host "$($DocLocation) contains more published artifacts than expected."
+        exit 1
     }
+
+    $DocsStagingDir = "$WorkingDirectory/docstaging"
+    $TempDir = "$WorkingDirectory/temp"
+
+    New-Item -ItemType directory -Path $DocsStagingDir
+    New-Item -ItemType directory -Path $TempDir
+
+    Expand-Archive -LiteralPath $PublishedDocs[0].FullName -DestinationPath $DocsStagingDir
+    $pkgProperties = VerifyPackages -pkgRepository "Nuget" -artifactLocation $DocLocation -workingDirectory $TempDir `
```

Comment:
Is DocLocation the correct location to pass as the artifactlocation? Would you pass PublicArtifactLocation?

Reply:
Why are we not just using what was already here and calling RetrieveReleaseTag?

Reply:
`Verify-Packages` and `RetrieveReleaseTag` takes in the same artifact location, the `artifactLocation` must include one .nuget file (no symbol one).

`$DocLocation` currently contains two .neget file, so it is not the right one to use.
Once we have $pkgProperties from VerifyPackages, we are able to get all package metadata, so there is no need to call RetrieveReleaseTag.

Reply:
VerifyPackages does extra work and needs things like the release sha which I don't think we care about for our purposes. I also think VerifyPackages returns back all the packages even the symbols packages similar to RetrieveReleaseTag and if not then we should make RetrieveReleaseTag work the same way. 

In either case I think the correct fix is to start ignoring the symbols package in the metadata parsing code.

Reply:
I agreed with you for the part of Verify-Packages done much more than RetrieveReleaseTag.
However, it seems extra burden to have packageId and packageVersion to parse out again as we did it in $parseNPMArifact already. Can we utilize the method in languageSetting.ps1?